src/main/webapp/map.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Maps Part One</title>
+  <link rel="stylesheet" href="/css/map.css">
+    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyCgr6Zutm1Os7q9V5Fi9N_g2poKcDG2G0A">
+    </script>
+    <script>
+      
+
+      // coordinates of homeless shelters in Washington
+      var shelters = [
```

Comment:
Ideally this information should be served via Java code and not hardcoded into the HTML. Is the plan to create servlet code in a followup change?

Reply:
So as of now I hardcoded it because this dataset only had files like .shp, .shx, .cpg, which I wasn't sure how to read and extract data I needed. It also had an .xml file but it contained a lot of unnecessary information. I've been having some trouble finding workable datasets.

Reply:
I was also looking for ways to import the data in the google maps documentation - and it seems to directly import the data through the html file. By servlet code, do you mean fetching the data using servlet logic?